Resources/Private/Partials/Poll/List/Table.html

```diff
@@ -16,16 +15,18 @@
         </tr>
         </thead>
         <tbody>
-        <f:if condition="{polls -> f:count()} > 0">
+
+        <f:if condition="{pagination.paginator.paginatedItems -> f:count()} > 0">
```

Comment:
IMHO a poll list without activated pagination will give an empty table here. Either add a pagination helper in Controller even if no pagination is set-up or adjust the template here to work with paginated and non-paginated lists.

Reply:
the pagination is now hidden, when there are only one page to be displayed.